src/core/opts.c

```diff
@@ -140,8 +140,10 @@ static error_t _bf_opts_parser(int key, char *arg, struct argp_state *state)
         break;
     case 'v':
         r = bf_verbose_to_str(arg, &opt);
-        if (r < 0)
+        if (r < 0) {
+            bf_info("valid --verbose options: [debug, bpf, bytecode]");
             return bf_err_r(EINVAL, "unknown --verbose option '%s'", arg);
```

Comment:
List the valid options from the error message directly, i.e. "unknown --verbose option '%s'. Valid options are ...". Otherwise the user might miss it and jump directly the the error message.

Reply:
Hi, changed it up, lmk if that's all good, also was looking for a quick tutorial in the docs (a "Quick Start" example??). I watched your talk here: [bpfilter: a BPF-based packet filtering framework](https://www.youtube.com/watch?v=fzaPEm4PXn0) and took some notes, I feel those could be useful for the docs but lmk

Reply:
A Quick Start section would definitely be valuable in the doc. Something like starting the daemon, using `bfcli` to define one or more filtering rules, and retrieve the counters (counters are fetched through `bpftool` for now, see #151). Let me know what you would like to see in the doc (feel free to create a new issue for this) and I'm happy for you to do it if you want to! :)